recipe/meta.yaml

```diff
@@ -7,11 +7,11 @@ package:
 
 source:
   - url: https://github.com/tensorflow/tensorflow/archive/refs/tags/v{{ version.replace(".rc", "-rc") }}.tar.gz
-    sha256: 9cec5acb0ecf2d47b16891f8bc5bc6fbfdffe1700bdadc0d9ebe27ea34f0c220
+    sha256: c729e56efc945c6df08efe5c9f5b8b89329c7c91b8f40ad2bb3e13900bd4876d
     patches:
       - patches/0001-loosen-requirements.patch
       - patches/0002-Add-additional-absl_synchronization-linkage-to-gRPC.patch
-      - patches/0003-Add-missing-any_invocable-target-to-absel-bazel.patch
+      # - patches/0003-Add-missing-any_invocable-target-to-absel-bazel.patch
```

Comment:
This has been made obsolete by https://github.com/tensorflow/tensorflow/commit/b34e129870f76ceb2daa92eedeae901d6540781e, but I didn't yet want to rebase out the patch, as it would further blow up the diff here.